### PR TITLE
Add support for EnergyPlus v22.2.0 with Ubuntu 22.04 base images

### PR DESCRIPTION
**Pull request by axelstudios**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -34,7 +35,7 @@ ENV SIMDATA_DIR=/var/simdata
 
 # Download
 RUN apt-get update \
-    && apt-get install -y ca-certificates curl libx11-6 libexpat1 python3 python3-pip \
+    && apt-get install -y ca-certificates curl libx11-6 libexpat1 python3 python3-pip $(["$UBUNTU_BASE" = "22.04"] && echo -n "libmd0") \
```
Comment (Myoldmopar) [inline Dockerfile:38]: You wanna leave this 10 minute effort in here for the future I suppose?

File: Dockerfile [hunk 2]
```diff
@@ -69,6 +70,13 @@ ONBUILD COPY --from=base \
     /usr/lib/x86_64-linux-gnu/libexpat.so* \
     /usr/lib/x86_64-linux-gnu/
 
+FROM ubuntu:22.04 as build_22.04
+ONBUILD COPY --from=base \
+    /usr/lib/x86_64-linux-gnu/libbsd.so* \
+    /usr/lib/x86_64-linux-gnu/libexpat.so* \
+    /usr/lib/x86_64-linux-gnu/libmd.so* \
+    /usr/lib/x86_64-linux-gnu/
```
Comment (Myoldmopar) [inline Dockerfile:78]: Interesting.

**Review by Myoldmopar — APPROVED** [review]
Seems good to me.  The manual pulling libraries from different versions is ... unexpected.  But if it works, it works.  I assume you want to leave the 22.04 conditional, so I'll mark it approved.

File: Dockerfile (hunk 1 above)
Comment (axelstudios) [reply Dockerfile:38]: Yep, until we determine it's not necessary.  And the conditional base image builds work really well

**Pull request merged**